src/goto-instrument/cover_basic_blocks.h

```diff
@@ -18,44 +18,84 @@ Author: Daniel Kroening
 
 class message_handlert;
 
-class cover_basic_blockst final
+class cover_blocks_baset
```

Comment:
Nit: move this into its own file

Reply:
I don't know, this is just an interface so very little code is added and this file is relatively small...